src/components/widgets/accounts-ratios/accounts-ratios.tmpl.html

```diff
@@ -86,7 +86,7 @@ <h4>Widget settings</h4>
 
         <div class="history chart-container" ng-show="widget.isHistoryMode">
           <div impac-chart draw-trigger="::drawTrigger.promise" deferred="::chartDeferred"></div>
-          <div class="legend">{{ widget.content.layout.ratio }}</div>
+          <div ng-show="allowXAxisLegend" class="legend">{{ widget.content.layout.ratio }}</div>
```

Comment:
Prefer `ng-if` over `ng-show`. `ng-show` is really only useful if you want to specifically want to keep an element in the DOM.